spk/bazarr/Makefile

```diff
@@ -0,0 +1,41 @@
+SPK_NAME = bazarr
+
+BAZARR_VERSION = 0.8.4.2
+
+SPK_VERS = $(BAZARR_VERSION)
+SPK_REV = 2
+
+SPK_ICON = src/bazarr.png
+
+REQUIRED_DSM = 6.1
```

Comment:
what is the reason to define a minimal DSM Version?

Reply:
None apart from the fact that since I was only testing it on that version.
I could put any version to be honest but then the packaging mechanism and the user logics would have to be checked, right ?
All the other packages I saw which were supporting older versions (than the 6, that is) seemed to have different logics to implement.

Reply:
IMHO we use REQUIRED_DSM when the spk explicitly depends on DSM features. For 6.1 (6.0 would be enought) this could be nginx, php configurations, minimal kernel, libc or TC compiler version etc.
As with `make all-supported` we officially support version 6.1 only (and arch-ppc853x-5.2) there is no need to use REQUIRED_DSM for other purpose.

Reply:
Done.